pos_hide_banknote_button/README.rst

```diff
@@ -0,0 +1,70 @@
+======================================
+Point of Sale - Disable Change Cashier
+======================================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-legalsylvain%2Fpos-lightgray.png?logo=github
+    :target: https://github.com/legalsylvain/pos/tree/12.0-pos_disable_change_cashier/pos_disable_change_cashier
+    :alt: legalsylvain/pos
+
+|badge1| |badge2| |badge3| 
+
+This module extends Odoo Point Of Sale features, allowing to disable
+the button that allow to switch cashiers.
```

Comment:
Mmm. Can you regenerate the README :)

Reply:
removed. let @OCA-git-bot generates a correct one. Thanks.